spec/syntax/heredoc_spec.rb

```diff
@@ -4,28 +4,42 @@
 
 describe 'Heredoc syntax' do
   describe 'binary' do
-    it 'with multiline content' do
+    it 'doc with multiline content' do
       expect(<<~EOF).to include_elixir_syntax('elixirDocString', 'foo')
         @doc """
         foo
         """
       EOF
     end
 
-    it 'escapes quotes unless only preceded by whitespace' do
+    it 'doc with sigil_S' do
+      expect('~S(foo)').to include_elixir_syntax('elixirDocString', 'foo')
```

Comment:
True. With `@doc` it would be a valid `elixirDocString`, but it's also not a heredoc, so I'm not sure it belongs in this test file. I haven't found any examples of docs being written that way. I have found some with `~S'''`, though. I'll modify the test.